.github/workflows/release.yml

```diff
@@ -1,124 +1,251 @@
-name: Build new release
+name: Release
 
 on:
   push:
-    branches:
-      - main
-
-permissions:
-  contents: write
-  packages: write
-  pull-requests: read
+    branches: ['main']
 
 jobs:
-  # Gets the build version and information about if this is a new version
-  get-version:
-    name: Get version
+  preconditions:
     runs-on: ubuntu-latest
     outputs:
-      is_new_version: ${{ steps.get_version.outputs.IS_NEW_VERSION }}
-      version: ${{ steps.get_version.outputs.VERSION }}
-      build_date: ${{ steps.get_version.outputs.BUILD_DATE }}
-      is_prerelease: ${{ steps.get_version.outputs.IS_PRERELEASE }}
+      repo_name: ${{ steps.repo_ids.outputs.REPO_NAME }}
+      org_name: ${{ steps.repo_ids.outputs.ORG_NAME }}
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - run: git fetch --tags
-      - run: |
-          sudo apt-get update
-          sudo apt-get install jq -y
-      - name: Get Build Version
-        id: get_version
-        run: ./scripts/check-version.sh
-        shell: bash
+      - name: Check token
+        run: |
+          if [ -z "${{ secrets.GITHUB_TOKEN }}"]; then
+            echo "Must provide a GITHUB_TOKEN secret in order to run release workflow"
+            exit 1
+          fi
+      - name: Get repository identifiers
+        id: repo_ids
+        run: |
+          REPO_NAME=$(echo "${{ github.event.repository.name }}" | tr '[:upper:]' '[:lower:]')
+          ORG_NAME=$(echo "${{ github.event.repository.owner.name }}" | tr '[:upper:]' '[:lower:]')
```

Comment:
no idea what this is doing

Reply:
It populates repo IDs and Org names for use throughout the rest of the workflow.